### PR TITLE
i18n: 🇵🇱

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

File: config/locales/pl.yml [hunk 1]
```diff
@@ -757,6 +758,16 @@ pl:
...
     prev: Poprzednia
     truncate: "&hellip;"
+  polls:
+    errors:
+      already_voted: Już zagłosowałeś(-aś) w tym głosowanie
+      duplicate_options: zawiera powtarzające się opcje
+      duration_too_long: jest zbyt odległa
+      duration_too_short: jest zbyt bliska
+      expired: To głosowanie już zakończyło się
+      over_character_limit: nie mogą składać się z więcej niż %{MAX} znaków
```
Comment (Aditoo17) [inline config/locales/pl.yml]:
```suggestion
      over_character_limit: nie mogą składać się z więcej niż %{max} znaków
```
%{MAX} was recently changed to %{max}, please change this line.

File: config/locales/pl.yml [hunk 2]
```diff
@@ -757,6 +758,16 @@ pl:
...
+      too_few_options: musi zawierać przynajmniej dwie opcji
+      too_many_options: nie może zawierać więcej niż %{MAX} opcji
```
Comment (Aditoo17) [inline config/locales/pl.yml]:
```suggestion
      too_many_options: nie może zawierać więcej niż %{max} opcji
```
See above.

**Review by Aditoo17 — CHANGES_REQUESTED** [review]

File: config/locales/pl.yml (hunk 2 above)
Comment (m4sk1n) [reply config/locales/pl.yml]: thanks

**Review by Aditoo17 — APPROVED** [review]

**Comment by m4sk1n** [comment]
tbh i’m unable to normalize it, could somebody do it for me?

**Comment by Gargron** [comment]
Can you submit the changes over Weblate instead?

**Comment by m4sk1n** [comment]
ok

**Comment by Aditoo17** [comment]
I saw that you added some strings which aren't in the `app/javascript/mastodon/locales/en.js` file yet. Might that be the cause?

**Comment by m4sk1n** [comment]
afaik `check-i18n` is for /config/locales/ files

**Pull request closed without merging**